pages/HomePage.js

```diff
@@ -0,0 +1,22 @@
+import BasePage from '../pages/BasePage';
+
+class HomePage extends BasePage {
+
+  //WebElements
+  get title(){ return $('#page-title') }
+
+  get userConf() { return $('img[alt="User Avatar"]')};
+
+  get logOut() {return $('a[href="/bank/logout"]')}
```

Comment:
Convention for naming entities tipically assign verbs to **functions/methods** and substantives to **attributes/variables**. This is a getter function, which is basically defining an attribute (but providing custom logic for retrieving it). Semantically, it is used as an attribute, therefore the name should refer to an object. In this case, this is a button, or menu item. Rename it to: **logOutButton** or something that points to an object instead of an action (in this case, simply **log out** is an action).

Names, in general, are one of the main tools we have at our disposal for creating readable, maintainable code.